src/App.js

```diff
@@ -1,19 +1,178 @@
 import React from 'react';
 import './App.css';
+import { ToDoList } from './components/ToDoList';
 
 import users from './api/users';
+import todos from './api/todos';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+const prepearedTodos = users.map(user => ({
+  ...user,
+  todo: todos.filter(todo => user.id === todo.userId),
+}));
+
+class App extends React.Component {
+  state = {
+    listOfTodos: [...prepearedTodos],
+    user: '',
+    todo: '',
+    inValidSelect: false,
+    inValidLength: false,
+    isTooLong: false,
+    containsSymbols: false,
+    containsWords: true,
+  }
+
+  changeInSelect(e) {
+    this.setState({
+      user: e.target.value,
+      inValidSelect: false,
```

Comment:
if I select a user, the select IS valid = it has value (so you need to change isValidSelect to true, cause false is the default value)